crawl-ref/source/mutation-data.h

```diff
@@ -2,6 +2,62 @@
 
 #include "tag-version.h"
 
+/*Mutation weights used when acquiring mutations. (Not exhaustive)
+Mutations are classified as good or bad. Good/bad mutations should
+total 100 weight. Zero weighted muts can go at the bottom.*/
+
+//Good Mutations.
+short MUT_IRIDESCENT_SCALES_WEIGHT = 6;
+short MUT_STRONG_WEIGHT = 5;
+short MUT_CLEVER_WEIGHT = 5;
+short MUT_AGILE_WEIGHT = 5;
+short MUT_MP_WANDS_WEIGHT = 5;
+short MUT_CLARITY_WEIGHT = 5;
+short MUT_WILD_MAGIC_WEIGHT = 5;
+short MUT_HORNS_WEIGHT = 4;
+short MUT_TALONS_WEIGHT = 4;
+short MUT_STRONG_WILLED_WEIGHT = 4;
+short MUT_ROBUST_WEIGHT = 4;
+short MUT_HOOVES_WEIGHT = 4;
+short MUT_ANTENNAE_WEIGHT = 3;
+short MUT_POISON_RESISTANCE_WEIGHT = 3;
+short MUT_HEAT_RESISTANCE_WEIGHT = 3;
+short MUT_COLD_RESISTANCE_WEIGHT = 3;
+short MUT_MUTATION_RESISTANCE_WEIGHT = 3;
+short MUT_EVOLUTION_WEIGHT = 3;
+short MUT_PASSIVE_MAPPING_WEIGHT = 2;
+short MUT_SHOCK_RESISTANCE_WEIGHT = 2;
+short MUT_REGENERATION_WEIGHT = 2;
+short MUT_ACUTE_VISION_WEIGHT = 2;
+short MUT_NOISE_DAMPENING_WEIGHT = 2;
+short MUT_CLAWS_WEIGHT = 2;
+short MUT_SHAGGY_FUR_WEIGHT = 2;
+short MUT_HIGH_MAGIC_WEIGHT = 2;
+short MUT_LARGE_BONE_PLATES_WEIGHT = 2;
+short MUT_THIN_SKELETAL_STRUCTURE_WEIGHT = 2;
+short MUT_STURDY_FRAME_WEIGHT = 2;
+short MUT_PASSIVE_FREEZE_WEIGHT = 1;
+short MUT_FANGS_WEIGHT = 1;
+short MUT_BEAK_WEIGHT = 1;
+short MUT_CAMOUFLAGE_WEIGHT = 1;
+
+//Bad Mutations.
+short MUT_READ_SAFETY_WEIGHT = 9;
+short MUT_DETORIORATION_WEIGHT = 9;
+short MUT_FRAIL_WEIGHT = 9;
+short MUT_LOW_MAGIC_WEIGHT = 9;
+short MUT_DEFORMED_WEIGHT = 8;
+short MUT_WEAK_WEIGHT = 8;
+short MUT_DOPEY_WEIGHT = 8;
+short MUT_CLUMSY_WEIGHT = 8;
+short MUT_BERSERK_WEIGHT = 7;
+short MUT_SCREAM_WEIGHT = 6;
+short MUT_SUBDUED_MAGIC_WEIGHT = 6;
+short MUT_DEVOLUTION_WEIGHT = 4;
+short MUT_INHIBITED_REGENERATION_WEIGHT = 3;
+short MUT_TELEPORT_WEIGHT = 3;
+short MUT_NO_POTION_HEAL_WEIGHT = 3;
```

Comment:
Thank you for doing the work of re-calculating and calibrating the weights!

It's somewhat unfortunate that mut_data is an array, so that mutations can't be grouped in a way that makes it easy to see group weights. However, since these weight numbers are not declared except in the mutation data structure, crawl style is not to create separate constants for each one. If you want to leave an easy-to-use summary of the good and bad total weights, a comment containing the table at the start of the array is the way to go.

If you want to be able to move the mutation_defs to group them, you'll need to refactor mut_data from an array to a map. Probably not worth the effort.